paper-button-behavior.d.ts

```diff
@@ -0,0 +1,47 @@
+/**
+ * DO NOT EDIT
+ *
+ * This file was automatically generated by
+ *   https://github.com/Polymer/gen-typescript-declarations
+ *
+ * To modify these typings, edit the source file(s):
+ *   paper-button-behavior.html
+ */
+
+/// <reference path="../polymer/types/polymer.d.ts" />
+/// <reference path="../iron-behaviors/iron-button-state.d.ts" />
+/// <reference path="paper-ripple-behavior.d.ts" />
+
+declare namespace Polymer {
+
+  interface PaperButtonBehavior {
+
+    /**
+     * The z-depth of this element, from 0-5. Setting to 0 will remove the
+     * shadow, and each increasing number greater than 0 will be "deeper"
+     * than the last.
+     */
+    readonly elevation: number|null|undefined;
+    hostAttributes: object|null;
+
+    /**
+     * In addition to `IronButtonState` behavior, when space key goes down,
+     * create a ripple down effect.
+     *
+     * @param event .
```

Comment:
is the space + dot intentional here and below?

Reply:
That's how it's documented in the source: https://github.com/PolymerElements/paper-behaviors/blob/master/paper-button-behavior.html#L66

Reply:
argh nvm, that's how it is in the sourcecode https://github.com/PolymerElements/paper-behaviors/blob/master/paper-button-behavior.html#L66